cmd/example-hook-sidecar/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM fedora:27
```

Comment:
if the example-hook-sidecar only needs libc (in golang) then you can just use `FROM scratch` which will make it pretty small